### DISCUSSION TITLE
The page path request error causes 404 pages to be displayed abnormally.

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#7039] [Q&A] [answered]
![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/8a2aef44-edfd-48f6-bd2b-c6e172cbd118)

I think the correct path should be: `https://ezqfnu.w1ndys.top/assets/javascripts/bundle.1e8ae164.min.js`

My source code file：https://github.com/w1ndys/easy-qfnu

My website: https://ezqfnu.w1ndys.top/

I'm deployed in Zeabur.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @W1ndys,
please don't repeat questions❗
I answered you in: 
- https://github.com/squidfunk/mkdocs-material/discussions/7015#discussioncomment-9034950

Your web server, where your files are deployed, needs to load the 404.html page which contains absolute paths starting with the base of your site_url. Zeabur loads another page, probably your home page that contains relative paths and the relative paths try to fetch the files relatively to the incorrect path.

This is out of scope for this discussion board. You need to contact Zeabur forums to get help.

**Reply by w1ndys** [reply]
Sorry, I know this is a repetitive question, but I have a doubt.
That is, the site_url I set is exactly the same as my actual domain name, which leads me to doubt the previous question.
I have sent a consultation to Zeabur and am waiting for their reply.

**Reply by w1ndys** [reply]
Zeabur has replied, and we all think that this is Zeabur's reason. They have started to deal with it, and I will give feedback again after the test is completed.

**Reply by w1ndys** [reply]
@kamilkrzyskow 
Update: Zeabur has fixed this problem.
> By the way, Zeabur did not support the deployment of Mkdocs at first. I discovered this platform and gave my opinions to the official staff. Then the official staff gave a one-click deployment end. If you want to put Zeabur's deployment plan in the official documentation If so, here is some information

https://m.youtube.com/watch?v=DYlpqWQgQEM
https://discord.com/channels/1060209568820494336/1227647766230990919
https://discord.com/channels/1060209568820494336/1225359021448499262
https://mkdocs.zeabur.app/
